src/main/java/io/github/raeperd/realworld/domain/user/User.java

```diff
@@ -21,8 +24,17 @@ public class User {
 
```

Comment:
Ensure that the 'passwordHash' column transformation is compatible with your database and that the cryptographic function used is secure and meets your security requirements.